frontend/src/app/projects/models/model-detail/model-detail.component.ts

```diff
@@ -22,9 +22,8 @@ import { ProjectService } from 'src/app/services/project/project.service';
   styleUrls: ['./model-detail.component.css'],
 })
 export class ModelDetailComponent implements OnInit, OnDestroy {
-  public gitModels: Array<GetGitModel> = [];
-  private _t4cModels = new BehaviorSubject<T4CModel[] | undefined>(undefined);
-  public t4cModels: T4CModel[] = [];
+  public gitModels: Array<GetGitModel> | undefined = undefined;
+  public t4cModels: T4CModel[] | undefined = undefined;
```

Comment:
```suggestion
  public gitModels?: GetGitModel[] = undefined;
  public t4cModels?: T4CModel[] = undefined;
```
I would definitely change the `array<...>` to `...[]`, since we typically use the `[]` notation (even directly below it). 
The `xy | undefined = ...` to `?: xy = ...` is more or less a preference thing, so both variants are valid imo.